tests/neg/i4496b.scala

```diff
@@ -0,0 +1,25 @@
+import scala.reflect.Selectable.reflectiveSelectable
+
+trait Foo1 { val a: Int }
+trait Foo2 { def a: Int }
+trait Foo3 { var a: Int }
+
+object TestStructuralVar {
+  type T0 = {var a: Int} // error
```

Comment:
Unrelated, but maybe this should be allowed and given the same semantics as having a getter/setter ?

Reply:
On second thoughts, maybe that's not worth it, since then it needs to be meaningful for all possible interpretation of structural types, nevermind.

Reply:
...but maybe the error message could suggest replacing the var by a val and a setter.